File: scala-spring-boot/src/test/java/com/yd/scala/helloscala/BaseTest.scala
package com.yd.scala.helloscala

import com.yd.scala.hello.StartApp
import org.junit.runner.RunWith
import org.springframework.boot.test.context.SpringBootTest
import org.springframework.test.context.junit4.SpringRunner

/**
  * @author created by Zeb灬D on 2019-12-24 10:59
  */
@RunWith(classOf[SpringRunner])
@SpringBootTest(classes = Array(classOf[StartApp]))
class BaseTest

File: scala-grpc/src/test/scala/com/yd/BaseTest.java
package com.yd;

/**
 * @author created by Zeb-D on 2019-08-19 11:39
 */

import com.yd.scala.grpc.GrpcApp;
import org.junit.runner.RunWith;
import org.springframework.boot.test.context.SpringBootTest;
import org.springframework.test.context.junit4.SpringRunner;

@RunWith(SpringRunner.class)
@SpringBootTest(classes = GrpcApp.class)
public class BaseTest {
}
